tests/cases/issues/issue_6651_export_sort.slint

```diff
@@ -0,0 +1,27 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-2.0 OR LicenseRef-Slint-Software-3.0
+
+import { B } from "../../helper_components/issue_6651_implicit_export.slint";
```

Comment:
Can you also import A to check it is indeed exported?

Reply:
Good idea.